README.md

```diff
@@ -91,6 +90,30 @@ Use the `ay-menu-button/dist/angular1.js` file:
 ```
 
 
+### Vue
+
+Use the `ay-menu-button/dist/vue.js` file:
+
+```javascript
+  import Vue from 'vue';
+  import AyMenuButton from 'ay-menu-button/vue';
```

Comment:
I think this needs to be `ay-menu-button/dist/vue`?

Reply:
updated!